push_test/pushclient.py

```diff
@@ -0,0 +1,335 @@
+import asyncio
+import base64
+import json
+import uuid
+import sys
+import traceback
+
+import aiohttp
+import websockets
+
+
+class PushException(Exception):
+    pass
+
+
+def log_msg(out=sys.stdout, **msg):
+    print(json.dumps(msg), file=out)
```

Comment:
Partly because this was actual, planned output by the program, rather than just filterable logging messages. Plus logging messages usually have extra information associated with them that would not be suitable for the app that will be reading the output.

That said, there's good reason to include traditional logging, and this function name may confuse, so will resolve.